lib/code-signing/code-signing-certificate.ts

```diff
@@ -97,23 +81,22 @@ export class CodeSigningCertificate extends cdk.Construct implements ICodeSignin
   /**
    * The ARN of the AWS Secrets Manager secret that holds the private key for this CSC
    */
-  public readonly privateKeySecretArn: string;
+  public readonly secretArn: string;
 
   /**
    * The ID of the version of the AWS Secrets Manager secret that holds the private key for this CSC
    */
-
-  public readonly privateKeySecretVersionId: string;
+  public readonly secretVersionId: string;
 
   /**
-   * The name of the AWS SSM parameter that holds the certificate for this CSC.
+   * The ARN of the AWS SSM Parameter that holds the certificate for this CSC.
    */
-  public readonly certificateParameterName: string;
```

Comment:
I thought the previous names were clearer. `parameter` and `secret` are the mechanisms in which you store these, not the actual intent

Reply:
Yeah but I still want to have a common interface for the PGPSecret and PrivateKey...

Reply:
Both have a "private part" and a "public part", no?